contrib/authors_update.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# See file LICENSE for terms.
+#
+
+set -eEu -o pipefail
+
+range="${1?Provide commit range like orig/v1.17..orig/v1.18}"
+
+if [ ! -w AUTHORS ]
+then
+    echo "AUTHORS file not accessible"
```

Comment:
`is not writable`

Reply:
fixed, but -w also covers the case where file does not exist